dagmc_stats/DagmcQuery.py

```diff
@@ -99,5 +142,81 @@ def calc_tris_per_vert(self, ignore_zero=True):
         if self._vert_data.empty:
             self._vert_data = self._vert_data.append(t_p_v_data)
         else:
-            self._vert_data.set_index('vert_eh').join(
-                self._vert_data.append(t_p_v_data).set_index('vert_eh'))
+            self._vert_data.set_index('vert_eh').join(self._vert_data.append(t_p_v_data).set_index('vert_eh'))
+
+    def update_tri_data(self, new_data):
```

Comment:
I think that this could be a private function instead. It's not useful for a user I think..